src/ObjCRuntime/RuntimeOptions.cs

```diff
@@ -45,17 +41,9 @@ static string ParseHttpMessageHandler (Application app, string? value)
 			switch (value) {
```

Comment:
The entire `#if MTOUCH || MMP || BUNDLER` block can be deleted now (and any corresponding code in mtouch/mmp/dotnet-linker as well).